### PR TITLE
refactor(benchpress): delete perf_util.ts and moved it's content to e2e_util.ts

### PR DESCRIPTION
**Pull request by wagnermaciel**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Currently we have our benchpress functions split into perf_util and e2e_util.

Issue Number: N/A

## What is the new behavior?
There is now only one file.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by wagnermaciel** [comment]
This change was made with the purpose of making the external api more similar to the internal one so that syncing benchpress drivers can be as easy as possible.

**Comment by wagnermaciel** [comment]
@jelbourn How do you feel about deleting the benchmarks_external folder?

- It hasn't been legitimately updated since [Mar 19, 2018](https://github.com/angular/angular/commits/eb8c5f709fd0da2b99ceba00d3704581aa8325e4/modules/benchmarks_external)
- It still uses `runClickBenchmark` which has not existed for a while. See [this example](https://github.com/angular/angular/blob/eb8c5f709fd0da2b99ceba00d3704581aa8325e4/modules/benchmarks_external/e2e_test/compiler_perf.ts#L9).

File: modules/e2e_util/e2e_util.ts [hunk 1]
```diff
@@ -5,13 +5,78 @@
...
  */
-
 /* tslint:disable:no-console  */
 import {browser} from 'protractor';
 import * as webdriver from 'selenium-webdriver';
 
+const nodeUuid = require('node-uuid');
+import * as fs from 'fs-extra';
+
+import {SeleniumWebDriverAdapter, Options, JsonFileReporter, Validator, RegressionSlopeValidator, ConsoleReporter, SizeValidator, MultiReporter, MultiMetric, Runner, StaticProvider} from '@angular/benchpress';
```
Comment (jelbourn) [inline modules/e2e_util/e2e_util.ts]: I would put all of the `import` statements before any `require` statements

File: modules/e2e_util/e2e_util.ts [hunk 2]
```diff
@@ -5,13 +5,78 @@
...
+
 declare var expect: any;
 
+// Note: Keep the `modules/benchmarks/README.md` file in sync with the supported options.
```
Comment (jelbourn) [inline modules/e2e_util/e2e_util.ts]: I would add a comment somewhere in this file that points out that it's important that `runBenchmark` and `openBrowser` are in the same file

**Review by jelbourn — COMMENTED** [review]
Seems like that folder could be deleted if it's not even building

File: modules/e2e_util/e2e_util.ts [hunk 3]
```diff
@@ -5,13 +5,80 @@
...
+import * as fs from 'fs-extra';
 import {browser} from 'protractor';
 import * as webdriver from 'selenium-webdriver';
 
+const nodeUuid = require('node-uuid');
+
 declare var expect: any;
 
+// Note: openBrowser, runBenchmark, and verifyNoBrowserErrors must be exported
+// from the same file to match our internal API.
```
Comment (jelbourn) [inline modules/e2e_util/e2e_util.ts]: nit: I would say "Google-internal" to be explicit

**Review by jelbourn — APPROVED** [review]
LGTM

**Review by mhevery — APPROVED** [review]

**Review by mhevery — CHANGES_REQUESTED** [review]
- [x]  either squash or use fixup! https://dev.to/koffeinfrei/the-git-fixup-workflow-386d. This is a single unit of work and should have single commit when merged.

**Comment by wagnermaciel** [comment]
@mhevery Done

**Review by mhevery — APPROVED** [review]

**Comment by mhevery** [comment]
LGTM, please add motivation to the commit msg.

**Comment by petebacondarwin** [comment]
The code that is touched by this PR has been considerably modified (e.g. caa4ab323506538fb812b4835f9c0ba317a08648)
So this PR is no longer viable.

**Pull request closed without merging**